docs/DELTA.md

```diff
@@ -86,4 +86,122 @@ if (auction?.status === 'EXECUTED') {
+  {chainId: SRC_CHAIN_ID, axios},
+  {
+    ethersProviderOrSigner: provider, // JsonRpcProvider
+    EthersContract: ethers.Contract,
+    account,
+  });
+  // for usage with different web3 provider libraries refer to the main [README](./README.md)
+```
+
+### 2. Check which tokens are available on the destination chain.
+
+A limited list of tokens are available in Across, the service facilitating crosschain bridging
+
+```ts
+const bridgeInfo = await deltaSDK.getBridgeInfo();
+
+const tokensAvailableForBridging = bridgeInfo[SRC_CHAIN_ID]?.[DEST_CHAIN_ID]
+```
+
+
+### 3. Request prices for a Token pair
+
+```ts
+const DAI_TOKEN = '0x6b175474e89094c44da98b954eedeac495271d0f';
+
+const USDC_TOKEN_ON_DEST_CHAIN = '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85'; // available for bridging on Optimism
+const amount = '1000000000000'; // in wei
+
+const deltaPrice = await deltaSDK.getDeltaPrice({
+  srcToken: DAI_TOKEN,
+  destToken: USDC_TOKEN_ON_DEST_CHAIN,
+  destChainId: DEST_CHAIN_ID, // required to get deltaPrice for a Crosschain Order
+  amount,
+  userAddress: account,
+  srcDecimals: 18,
+  destDecimals: 18,
+  // partner: "..." // if available
+});
+```
+
+
+### 4. Approve srcToken for DeltaContract
+
+```ts
+const tx = await deltaSDK.approveTokenForDelta(amount, DAI_TOKEN);
+await tx.wait();
+```
+
+Alternatively sign Permit (DAI or Permit1) or Permit2 TransferFrom with DeltaContract as the verifyingContract
+
+```ts
+const DeltaContract = await deltaSDK.getDeltaContract();
+
+// values depend on the Permit type and the srcToken
+const signature = await signer._signTypedData(domain, types, message);
+```
+
+See more on accepted Permit variants in [Velora documentation](https://developers.velora.xyz/api/velora-api/velora-delta-api/build-a-delta-order-to-sign#supported-permits)
+
+
+### 5. Sign and submit a Delta Order
+
+```ts
+// calculate acceptable destAmount
+const slippagePercent = 0.5;
+  const destAmountAfterSlippage = (
+    +deltaPrice.destAmount *
+    (1 - slippagePercent / 100)
+  ).toString(10);
+
+const deltaAuction = await deltaSDK.submitDeltaOrder({
+  deltaPrice,
+  owner: account,
+  // permit: "0x1234...", // if signed a Permit1 or Permit2 TransferFrom for DeltaContract
+  srcToken: DAI_TOKEN,
+  destToken: USDC_TOKEN,
+  srcAmount: amount,
+  destAmount: destAmountAfterSlippage, // minimum acceptable destAmount
+  destChainId: DEST_CHAIN_ID, // required to get construct a Crosschain Order
+  beneficiary: anotherAccount, // if need to send destToken to another account on destChain
+  isBeneficiaryContract: false, // whether the beneficiary on destChain is a smart contract
+  // bridge, // user-constructed Bridge object for Crosschain Orders
+});
+```
+
+To construct a Crosschain Delta Order it is required to either:
+* construct Bridge object. Refer to [documentation](https://developers.velora.xyz/api/velora-api/velora-delta-api/build-a-delta-order-to-sign#sign-an-order-cross-chain) for how to do that.
```

Comment:
is there a plan to integrate bridge params handling?

Reply:
Literally the next line says that it is handled by SDK if only the user gives isBeneficiaryContract param

Reply:
If you think it's not clear, feel free to reword

Reply:
Reword all you like, but I believe a Boolean better congrats the intent. 
beneficiaryContract implies address. Even something like beneficiaryType would be useless in vanilla JS, for example, always need TS to guide the correct value